doc/FORTANIX-SGX-ABI.md

```diff
@@ -2,6 +2,21 @@
 
 This document describes the ABI of SGX enclaves built using `libenclave`.
 
+## Enclave runner compatibility
```

Comment:
This is not related to the runner. It's the elf2sgxs conversion

Reply:
updated, thanks